Sources/TokamakCore/Modifiers/TextFieldStyle.swift

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 06/30/2020.
+//
+
+public protocol TextFieldStyle {}
+
+public struct DefaultTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+public struct PlainTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+public struct RoundedBorderTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+public struct SquareBorderTextFieldStyle: TextFieldStyle {
+  public init() {}
+}
+
+enum TextFieldStyleKey: EnvironmentKey {
+  static var defaultValue: TextFieldStyle = DefaultTextFieldStyle()
+}
+
+extension EnvironmentValues {
+  var textFieldStyle🔒: TextFieldStyle {
```

Comment:
Sorry, I'm not sure I'm following, why not just name it `textFieldStyle`? It's an internal symbol anyway, users won't have access to it, as far as I understand.

Reply:
I don’t think people would be able to create a custom `textFieldStyle` environment variable if this private one had that name. Not sure that’s an issue though, but that was my reasoning.

Reply:
They totally would be able to, when symbol names are compiled down to machine code (or WebAssembly for that matter) [they're mangled](https://mikeash.com/pyblog/friday-qa-2014-08-15-swift-name-mangling.html) with module names prefixing them, so even public symbols with the same name don't collide. Hope that makes sense 🙂